src/Administration/Resources/administration/src/app/mixin/listing.mixin.js

```diff
@@ -71,10 +71,18 @@ Mixin.register('listing', {
                 delete params.criteria;
             }
 
-            this.$router.push({
+            const route = {
```

Comment:
Since we are always in the listing here, I think you can generally replace the route:
```           
this.$router.replace({
                name: this.routeName,
                query: params
            });
```
This way the whole page has only entry in the history.

Reply:
I was thinking about this, especially because the current implementation has another bug: using the browser navigation in a listing does not updated the results, even though the query parameters in the URL change when "going back".

I came to the following conclusion: When the `page` parameter changes, a new route should be pushed, because in that case by going back in the browser history the user would expect to go back to the previously displayed page. For any other parameters (`sort`, `filter` etc.) a replace is probably more fitting.

Do you have any thoughts on this?

Reply:
Yes, that's exactly would I thought. We should only add a history entry to the browser if the page is changing. Filtering or sorting should not add any history entries.

You could add a check if the page has changed:
if true -> push a new history entry
false -> replace the history

I think this is possible by checking the route name.
You can check this.routeName against the route in the this.$router.....